recipes/zbar/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "zbar" %}
+{% set version = "0.10" %}
+{% set sha256 = "234efb39dbbe5cef4189cc76f37afbe3cfcfb45ae52493bfe8e191318bdbadc6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://sourceforge.net/projects/{{name}}/files/{{name}}/{{version}}/{{name}}-{{version}}.tar.bz2
+  sha256: {{ sha256 }}
+  patches:
+    - 0001-Add-cstddef-header-to-zbar.h-for-NULL.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - libjpeg-turbo
+  run:
+    - libjpeg-turbo
+
+test:
+  commands:
+    - bash -c "test -f $PREFIX/lib/libzbar.a"  # [unix]
+    - bash -c "test -f $PREFIX/lib/libzbar.so"  # [linux]
+    - bash -c "test -f $PREFIX/lib/libzbar.dylib"  # [osx]
+    - bash -c "test -d $PREFIX/include/zbar && test -f $PREFIX/include/zbar/ImageScanner.h"  # [unix]
+
+about:
+  home: http://github.com/simplejson/simplejson
```

Comment:
I think this should either be http://zbar.sourceforge.net/ or https://github.com/ZBar/ZBar